app/routes/events/components/EventList.js

```diff
@@ -124,6 +124,7 @@ type EventListProps = {
   fetchMore: () => Promise<*>
 };
 
+// $FlowFixMe
```

Comment:
Hmmmmm :thinking:
![screenshot_20181025-010700](https://user-images.githubusercontent.com/14221489/47466690-b398e780-d7f2-11e8-9d09-62b308eb99f6.jpg)

Reply:
https://github.com/webkom/lego-webapp/blob/990f636d1694e5df6b6907cf0a9f7f988e940a00/.flowconfig#L24
:tm:

Reply:
This isn't even my code 👎 
`() => (Not my code )=> (Not my mistake) => (Not my fault) => (Not my job)`